invenio_accounts/ext.py

```diff
@@ -93,6 +93,19 @@ def monkey_patch_flask_security():
             recoverable.hash_password = hash_password
             registerable.hash_password = hash_password
 
+        # Disable remember me cookie generation as it does not work with
+        # session activity tracking (a remember me token will bypass revoking
+        # of  a session).
+        def patch_do_nothing(*args, **kwargs):
+            pass
+        LoginManager._set_cookie = patch_do_nothing
+
+        # Disable loading user from header because we want to be sure we
+        # can load user only through the login form.
+        def patch_reload_anonym(self, *args, **kwargs):
+            self.reload_user()
+        LoginManager._load_from_header = patch_reload_anonym
```

Comment:
Same thing here. I think we should likely also patch ``LoginManager._load_from_request``?